### PR TITLE
Tweaked navbar

### PR DESCRIPTION
**Pull request by udayk8139**
* Added text-shadow to all the header text so it's readable when images are below.
* Changed the navbar color to FCC's primary colour, and used white to highlight the navigation menu and the active elements.

### CODE REVIEW

File: css/tweak.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.navbar-custom .nav li a:hover,
+.navbar-custom .nav li a:focus {
+  color: #FFF;
+  outline: none;
+
+}
+
+.navbar-custom .navbar-nav > .active > a:hover,
+.navbar-custom .navbar-nav > .active > a:focus {
```
Comment (vramana) [inline css/tweak.css]: Do you need selector of this complexity? Can't you make it any shorter?

Comment (udayk8139) [inline css/tweak.css]: Actually, yes I need it. 
If you just want it really small, I can simply add some more class and then it's done.

Comment (vramana) [inline css/tweak.css]: This rule seems redundant. Compare the styles in the devtools. Am I missing something?
image

Comment (udayk8139) [inline css/tweak.css]: What do you mean "Reductant" ?

Comment (vramana) [inline css/tweak.css]: It's redundant btw. Your code adds
```css
.some-selector {
  color: white;
  background-color: #307701;
}
```
Here is what's already present
```css
.some-other-selector {
  color: #fff;
  background-color: #307701;
}
```
I am telling what's the point of adding the same css rules.

Comment (udayk8139) [inline css/tweak.css]: Oh that, I'll just remove it ;3 Idk when I did that step being absent minded...

**Pull request merged**